src/styles/auto-prefix.js

```diff
@@ -20,18 +20,29 @@ export default {
     let prefixer = prefixers[userAgent];
     // None found, create a new instance
     if (!prefixer) {
-      prefixer = new InlineStylePrefixer(userAgent);
+      prefixer = new InlineStylePrefixer({userAgent: userAgent});
```

Comment:
@alitaheri, Yeah that's an interesting consideration. I'm split too but would probably prefer a rule that enforces one way or the other...

Just for reference, here is the rule: https://github.com/eslint/eslint/blob/master/docs/rules/object-shorthand.md
```
object-shorthand: [2, 'always']
```